MonoGame.Framework/Graphics/Viewport.cs

```diff
@@ -172,11 +173,42 @@ public Viewport(int x, int y, int width, int height)
 		    this.minDepth = 0.0f;
 		    this.maxDepth = 1.0f;
 		}
-		
+
+        /// <summary>
+        /// Creates a new instance of <see cref="Viewport"/> struct.
+        /// </summary>
+        /// <param name="x">The x coordinate of the upper-left corner, of the view bounds, in pixels.</param>

```

Comment:
The commas are not necessary here.

Also use this same description for the X/Y property as well.